src/test/java/com/puppycrawl/tools/checkstyle/checks/blocks/RightCurlyCheckTest.java

```diff
@@ -260,6 +260,18 @@ public void testAloneOrSingleLine() throws Exception {
     }
 
     @Test
+    public void testLambda() throws Exception {
```

Comment:
The test name should explain what it tests. Maybe test can be named as 'testRightCurlyIsAloneInLambda'.